examples/chip/gpio.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+from os import uname
+from socket import gethostname
+from time import sleep
+import sys
+import cloud4rpi
+import chip
+import CHIP_IO.GPIO as GPIO  # pylint: disable=F0401
+
+# Put your device token here. To get the token,
+# sign up at https://cloud4rpi.io and create a device.
+DEVICE_TOKEN = '__YOUR_DEVICE_TOKEN__'
+
+# Constants
+POLL_INTERVAL = 0.5  # 500 ms
+DATA_SENDING_INTERVAL = 30  # seconds * POLL_INTERVAL
+DIAG_SENDING_INTERVAL = 60  # seconds * POLL_INTERVAL
+
+
+def P0_control(value):
+    GPIO.output('XIO-P0', value)
+    return GPIO.input('XIO-P0')
+
+GPIO.setup('XIO-P0', GPIO.OUT)
+
+
+def main():
+    # Put variable declarations here
+    variables = {
+        'XIO-P0': {
+            'type': 'bool',
+            'value': False,
+            'bind': P0_control
+        }
+    }
+
+    # Put system data declarations here
+    diagnostics = {
+        'IP Address': chip.ip_address,
+        'Host': gethostname(),
+        'Operating System': " ".join(uname()),
```

Comment:
Or maybe it's better to change the raspberrypi examples? This code fixes things I don't like in raspberrypi examples and propose this to be changed there also.